src/components/calculator.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+class CalcUserInterface extends React.Component { // eslint-disable-line
+  render() {
+    return (
+      <section>
+        <div className="container">
+          <div className="header">0</div>
+          <div className="calc-button">
+            <button type="button">Ac</button>
+            <button type="button">+/-</button>
+            <button type="button">%</button>
+            <button type="button" className="box-color">÷</button>
+            <button type="button">7</button>
+            <button type="button">8</button>
+            <button type="button">9</button>
+            <button type="button" className="box-color">x</button>
+            <button type="button">4</button>
+            <button type="button">5</button>
+            <button type="button">6</button>
+            <button type="button" className="box-color">-</button>
+            <button type="button">1</button>
+            <button type="button">2</button>
+            <button type="button">3</button>
+            <button type="button" className="box-color">+</button>
+            <button type="button" className="zero">0</button>
+            <button type="button">.</button>
+            <button type="button" className="box-color">=</button>
+          </div>
```

Comment:
- [ ] Great work implementing the design :clap: However it is requested in the project requirement that the `Calculator.js` should have at least **1 child component**. Therefore I suggest you create a new component for these buttons and import it in the `Calculator.js` component to follow the project [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md)

Reply:
- [ ] As mentioned in the previous review, It is requested in the project requirement that the `Calculator.js` component should have at least **1 child component**(see image). Therefore, please create a new component for these `button` tags and import the new created component as the child for the `calculator.js` component to follow the project requirement.
![Screenshot from 2023-08-10 13-40-20](https://github.com/obibaadoma/math-magic/assets/91142432/f421d812-efde-48bd-afe8-0506628c2ed6)